index.js

```diff
@@ -55,15 +55,66 @@ const sendEmail = async ({
       expiredKeys,
     }),
   }
-  sgMail.send(msg)
+  //sgMail.send(msg)
 }
 
+const getKeyDetails = async ({
+  keyNames
+}) => {
+  const keyInfoFileContents = await readFileAsync(join(__dirname, './keys.info'))
```

Comment:
Can we perhaps add a `keys.info.example` file to this repo and check it in so future devs can reference that?

Reply:
Is this file `keys.info` or `keysinfo.json` (see `.gitignore`)? I think if it's a json file the file should have a `.json` extension 😄